### ISSUE TITLE
dokka {includeNonPublic} not working as documented

### ISSUE THREAD

**Issue by JWT007** [Kotlin/dokka#226]
According to the README.md I should be able to declare the following:

```
dokka {
  ...
  includeNonPublic = false
}
```

However, 'includeNonPublic' is not a valid parameter at this level.  

It is only defined in the packageOptions (and declaring it there results in a serialization error but I think there is another ticket for that issue).

**Comment by paragones** [comment]
I am also having the same problem

**Comment by paragones** [comment]
this is the same as 

    reportNotDocumented = true

**Comment by ColinHebert** [comment]
Those changes have been merged in master but we're waiting on @semoro to release a new version of the plugin to have access to those options.

**Comment by semoro** [comment]
Fixed in [0.9.16](https://github.com/Kotlin/dokka/releases/tag/0.9.16)
